mail_filter_adressee_by_contact/models/mail_compose_message.py

```diff
@@ -3,6 +3,7 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 from odoo import api, fields, models
+from odoo.osv import expression
 
 FILTERED_MODEL = [
```

Comment:
Not needed if apply the other suggestion

Reply:
I applied yours suggestions but don't understand how to do this one. On my point of view, I need "expression" to complete the domain,  depending of the model. no?

Reply:
It was an error :S